src/settings/routes/SupplementaryPropertiesConfigRoute.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { get } from 'lodash';
 import { stripesConnect } from '@folio/stripes/core';
-import SupplementaryPropertiesConfigForm from '../components';
+import SupplementaryPropertiesConfigForm from '../components/SupplementaryPropertiesConfigForm';
```

Comment:
Please add a new `index.js` file at the top level within `src/settings/components` and export the `SupplementaryPropertiesConfigForm` from there. That way this could be imported as 
```suggestion
import SupplementaryPropertiesConfigForm from '../components';
```

Reply:
Hm. I get it working with 
`import { SupplementaryPropertiesConfigForm } from '../components';`
as it's a named import?

Reply:
Sounds good